bin/seacan

```diff
@@ -76,19 +76,56 @@ package App::Seacan {
         my $cpanm_command = join_path( $self->seacan_perlbrew_root, "bin", "cpanm");
         my $perl_command = $self->seacan_perl;
 
+        my $dest = $self->config->{seacan}{app}
+            || join_path($self->config->{seacan}{output}, "app", $self->config->{seacan}{app_dir_name});
+
         $, = " ";
-        system($perl_command, $cpanm_command, "--notest", "-L", join_path($self->config->{seacan}{output}, "local"), "--installdeps", $self->config->{seacan}{app} ) == 0 or die $!;
+        system($perl_command, $cpanm_command, "--notest", "-L", join_path($self->config->{seacan}{output}, "local"), "--installdeps", $dest ) == 0 or die $!;
+    }
+
+    sub check_config {
+        my $self = shift;
+        my $conf_seacan = $self->config->{seacan};
+        my $conf_perl   = $self->config->{perl};
+
+        $conf_perl->{version} or die "conf <perl> must contain 'version'\n";
+
+        $conf_seacan->{output} or die "conf <seacan> must contain 'output'\n";
+
+        $conf_seacan->{app_name} or die "conf <seacan> must contain 'app_name'\n";
+
+        $conf_seacan->{app} or $conf_seacan->{app_github}
+            or die "conf <seacan> must contain either 'app' or 'app_github'\n";
+
+        if ( $conf_seacan->{app_github} ) {
```

Comment:
sure, but we'd still need to check we have 'app'. the validation is needed regardless of the git related change.